.github/workflows/mirror.yml

```
@@ -9,9 +9,10 @@ on:
 jobs:
     to_github:
         runs-on: ubuntu-18.04
+        if: github.repository_owner == 'pimcore'
```

```suggestion
        if: ${{ github.repository == 'pimcore/pimcore' }}
```
I think we can make this even more restrictive as we do here as well: https://github.com/pimcore/pimcore/blob/2320fe9185aa164296aa5e71815cb5feb1766fcd/.github/workflows/lts-repo-sync.yml#L11